Services/CO.CDP.Organisation.WebApi/UseCase/UpdateOrganisationUseCase.cs

```diff
@@ -31,6 +31,20 @@ public async Task<bool> Execute((Guid organisationId, UpdateOrganisation updateO
                 organisation.Name = updateObject.OrganisationName;
                 break;
 
+            case OrganisationUpdateType.Roles:
+                if (updateObject.Roles == null || !updateObject.Roles.Any())
+                {
+                    throw new InvalidUpdateOrganisationCommand("Missing roles.");
+                }
+
+                if (!updateObject.Roles.All(role => Enum.IsDefined(typeof(PartyRole), role)))
+                {
+                    throw new InvalidUpdateOrganisationCommand("One or more roles are invalid.");
+                }
```

Comment:
Is this a valid scenario? I'd assume the `updateObject` would not be built if request contained a value that's not part of the enum.

Show's we're missing tests :)

Reply:
No probably not. I actually tried to write a test for this and literally couldn't because of the way it was all defined. So I'll take this out.

Reply:
It's gone